### PR TITLE
Add custom media_extensions support

### LINKED ISSUE #301: [FR] Support more filetypes with telescope-media-files integrtion
**Please confirm**
- [x] I am running the latest version of this plugin

**Is your feature request related to a problem? Please describe.**
I'm using telescope-media-files to preview images, some file types are not listed it telescope list (webp, svg), which supported by telescope-media-files. Is possible to configure these filetypes?

**Describe the solution you'd like**
Support more file-types by default? https://github.com/renerocksai/telekasten.nvim/compare/main...Inom-Turdikulov:telekasten.nvim:main

### PR DESCRIPTION
**Pull request by Inom-Turdikulov**
Customize insert image and preview image files list.

## Type of change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Code quality improvements to existing code or addition of tests
- [ ] Documentation update

## Additional information

This is useful to add optional filetypes into filtered list (for example telescope-media-files optionally supporting svg preview).
With this solution user can control which media files he want to see in telescope windows.

- This PR fixes or closes issue: fixes #301
- This PR is related to issue: #301

## Checklist

- [x] I am running the **latest** version of the plugin.
- [x] The code change is tested and works locally (preview, media preview, insert link).
- [x] There is no commented out code in this PR.
- [x] The code has been formatted using Stylua (a `.stylua.toml` file is provided)
- [ ] The code has been checked with luacheck (a `.luacheckrc` file is provided)
- [ ] The `README.md` has been updated according to this change. No configuration section?
- [x] The `doc/telekasten.txt` helpfile has been updated according to this change.

### CODE REVIEW

**Comment by lambtho12** [comment]
Is there a reason to expose this as an option? This seems a bit unnecessary and cumbersome. Why not just expanding the hard-coded list to add the most common image extensions?

**Comment by Inom-Turdikulov** [comment]
Yes, main here few reasons (at least for me):

For example `Telekasten browser_media` is used to generate media files list and for example my preview plugin is not supporting svg files and I just don't want to see them here (lets assume I have a lot of svg files). Or for example I want to preview some rare file-type like .tiff, but it's not in a list. Here a lot of combinations.

I think chafa (which used by telescope-media-files plugin) support a lot file-types (probably anything which imagemagic support, `magick identify -list format` gives 302 file formats) and managing this file-types from configuration can bu useful.
image

File: lua/telekasten.lua [hunk 1]
```diff
@@ -148,6 +148,17 @@ local function defaultConfig(home)
...
+        -- files which will be aviable in insert and preview images list
+        media_extensions = {
+            ".png",
+            ".jpg",
+            ".bmp",
+            ".gif",
+            ".pdf",
+            ".mp4",
+            ".webm",
+            ".webp",
```
Comment (lambtho12) [inline lua/telekasten.lua:160]: You put `.webp` in the default configuration, but not in the documentation.
Likewise, you put `.svg` in the documentation but not in the configuration.

Could you sort this out so the documentation matches exactly the default config?

**Review by lambtho12 — CHANGES_REQUESTED** [review]
The doc should reflect exactly the default config.

I am fine with adding svg and webp to the default config, but then the doc should reflect it as well.

**Comment by lambtho12** [comment]
Thanks for explaining your implementation. I am fine with having it as an option then. Just address the review so the doc matches the default config and I will merge.

File: lua/telekasten.lua (hunk 1 above)
Comment (Inom-Turdikulov) [reply lua/telekasten.lua:160]: Yes, I updated it

**Comment by lambtho12** [comment]
Thank you !

**Pull request merged**